### PR TITLE
2016 02 mn set up euth proposal

### PR DESCRIPTION
**Pull request by MagdaN**
This adds the pc proposal to euth. To get this running run the following commands

bin/import_resources etc/development.ini src/adhocracy_euth/adhocracy_euth/resources/test_idea_collection/resources.json

bin/set_workflow_state etc/development.ini /opin/idea_collection/ announce participate

Browse to: http://localhost:6551/r/opin/idea_collection/

### CODE REVIEW

**Comment by xi** [comment]
I would rather move this code to core instead of duplicating it.

**Comment by MagdaN** [comment]
I would rather do that later in a different PR

File: src/euth/euth/static/js/Packages/Euth/IdeaCollection/Module.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import * as AdhEuthIdeaProposalModule from "./Proposal/Module";
+import * as AdhEuthIdeaContextModule from "./Context/Module";
+import * as AdhEuthIdeaWorkbenchModule from "./Workbench/Module";
+
+export var moduleName = "euthIdeaCollection";
```
Comment (xi) [inline src/euth/euth/static/js/Packages/Euth/IdeaCollection/Module.ts]: all our modules should have the `adh` prefix

File: src/euth/euth/static/js/Packages/Euth/IdeaCollection/Context/Module.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import * as AdhTopLevelStateModule from "../../../TopLevelState/Module";
+
+import * as AdhEuthWorkbenchModule from "../Workbench/Module";
+
+import * as AdhEmbed from "../../../Embed/Embed";
+import * as AdhResourceArea from "../../../ResourceArea/ResourceArea";
+
+import * as AdhEuthWorkbench from "../Workbench/Workbench";
+
+import RIPeuthProcess from "../../../../Resources_/adhocracy_euth/resources/idea_collection/IProcess";
```
Comment (xi) [inline src/euth/euth/static/js/Packages/Euth/IdeaCollection/Context/Module.ts]: is the "P" in "Peuth" still left from "Pcompass"? (also in some other places)

**Comment by xi** [comment]
If you really want to we can merge this. Still I am not too happy with duplicating 900 lines of code.

File: src/euth/euth/static/js/Packages/Euth/IdeaCollection/Context/Module.ts (hunk 2 above)
Comment (MagdaN) [inline src/euth/euth/static/js/Packages/Euth/IdeaCollection/Context/Module.ts]: I thought they with RIP - is the naming documented somewhere?

**Comment by MagdaN** [comment]
Well this is up to you as I don't really have time to work on this anymore

File: src/euth/euth/static/js/Packages/Euth/IdeaCollection/Context/Module.ts (hunk 2 above)
Comment (xi) [inline src/euth/euth/static/js/Packages/Euth/IdeaCollection/Context/Module.ts]: No, its just a convention. R/S is short for Resource/Sheet. For example, there is both an `IComment` resource and an `IComment` sheet. So we differentiate by calling them `RIComment`/`SIComment`. Feel free to add some documentation if you think that helps.

Comment (MagdaN) [inline src/euth/euth/static/js/Packages/Euth/IdeaCollection/Context/Module.ts]: How can I document a convention that I don't know? Where does the 'I' stand for?

**Pull request merged**